tests/IBTK/samraidatacache_01_2d.mpirun=1.output

```diff
@@ -0,0 +1,165 @@
+number of patch data indices before registering double precision vars:2
+number of patch data indices before registering single precision vars:11
+number of patch data indices before registering integer vars:20
+cloning pass 0
+number of patch data indices before cloning double precision vars:29
+number of patch data indices before cloning single precision vars:38
+number of patch data indices before cloning integer vars:47
+number of patch data indices after cloning all vars:56
+cloning pass 1
+number of patch data indices before cloning double precision vars:56
+number of patch data indices before cloning single precision vars:56
+number of patch data indices before cloning integer vars:56
+number of patch data indices after cloning all vars:56
+number of patch data indices after removing all cloned vars:56
+number of patch data indices after the cache goes out of scope:56
+
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+Printing PatchDescriptor<DIM> state ...
+this = 0x7f9cd3c1de20
```

Comment:
I don't know where we printed `this` but thats not going to be reproducible.

Reply:
Uh oh. That is just what the patch descriptor prints out when you ask it to output its state.

Reply:
I made my own routine to do this.